src/api/media/MediaControlConfiguration.ts

```diff
@@ -27,20 +27,14 @@ export interface MediaControlConfiguration {
   /**
    * The amount of seconds the player will skip forward.
    *
-   * @defaultValue 5
-   *
-   * @remarks
-   * <br/> - This property only applies to Web and Android.
+   * @defaultValue 5 on Web and android, 15 on iOS.
```

Comment:
This might look a bit weird, but for iOS the default was 15 and I would like to prevent using a different default value suddenly. It would affect customer applications.